compiler/src/parsing/parser.messages

```diff
@@ -6026,7 +6098,7 @@ Expected `(` followed by an expression for the while loop condition.
 
 program: MATCH LPAREN FLOAT32 RPAREN LBRACE NUMBER_INT EOL WHILE
 ##
-## Ends in an error in state: 777.
+## Ends in an error in state: 591.
```

Comment:
Looks like you regenerated all of the error states (which we really need to do) but it'd be a bit better if we did that in  a separate PR. I think I counted three new programs here; could you revert this file and add just those three changes and we'll regenerate in another PR? 🙏